components/endpoints-dashboard/src/endpoints-dashboard.module.js

```diff
@@ -3,30 +3,54 @@
 
   angular
     .module('endpoints-dashboard', [])
+    .constant('endpointsDashboardDisabledKey', 'endpointsDashboardDisabled')
     .run(register);
 
-  function register($q, $state, modelManager, appEventService, appUtilsService) {
-    return new EndpointsDashboard($q, $state, modelManager, appEventService, appUtilsService);
+  function register($q, $state, modelManager, appEventService, appUtilsService,
+                    endpointsDashboardDisabledKey, appLoggedInService) {
+    return new EndpointsDashboard($q, $state, modelManager, appEventService, appUtilsService,
+      endpointsDashboardDisabledKey, appLoggedInService);
   }
 
-  function EndpointsDashboard($q, $state, modelManager, appEventService, appUtilsService) {
+  function EndpointsDashboard($q, $state, modelManager, appEventService, appUtilsService, endpointsDashboardDisabledKey,
+                              appLoggedInService) {
     var initialized = $q.defer();
 
-    function init() {
-      return initialized.promise;
-    }
+    var config = {};
```

Comment:
Remove config object

change updateConfig to be isDashboardDisabled() that returns the value